2021-04-25 23:32:05,861 INFO    MainThread:1768 [wandb_setup.py:_flush():69] setting env: {}
2021-04-25 23:32:05,862 INFO    MainThread:1768 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-25 23:32:05,862 INFO    MainThread:1768 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\Henrik_Brådland\Google Drive\Studier\Semester 10\IKT-442\wandb\run-20210425_233205-1hl2pnjp\logs\debug.log
2021-04-25 23:32:05,862 INFO    MainThread:1768 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\Henrik_Brådland\Google Drive\Studier\Semester 10\IKT-442\wandb\run-20210425_233205-1hl2pnjp\logs\debug-internal.log
2021-04-25 23:32:05,862 INFO    MainThread:1768 [wandb_init.py:init():369] calling init triggers
2021-04-25 23:32:05,863 INFO    MainThread:1768 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-25 23:32:05,863 INFO    MainThread:1768 [wandb_init.py:init():418] starting backend
2021-04-25 23:32:05,863 INFO    MainThread:1768 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn, using: spawn
2021-04-25 23:32:05,870 INFO    MainThread:1768 [backend.py:ensure_launched():123] starting backend process...
2021-04-25 23:32:05,930 INFO    MainThread:1768 [backend.py:ensure_launched():127] started backend process with pid: 9392
2021-04-25 23:32:05,931 INFO    MainThread:1768 [wandb_init.py:init():423] backend started and connected
2021-04-25 23:32:05,933 INFO    MainThread:1768 [wandb_init.py:init():465] updated telemetry
2021-04-25 23:32:05,933 INFO    MainThread:1768 [wandb_init.py:init():484] communicating current version
2021-04-25 23:32:06,764 INFO    MainThread:1768 [wandb_init.py:init():489] got version response 
2021-04-25 23:32:06,764 INFO    MainThread:1768 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-25 23:32:07,035 INFO    MainThread:1768 [wandb_init.py:init():522] starting run threads in backend
2021-04-25 23:32:08,766 INFO    MainThread:1768 [wandb_run.py:_console_start():1524] atexit reg
2021-04-25 23:32:08,767 INFO    MainThread:1768 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.WRAP
2021-04-25 23:32:08,767 INFO    MainThread:1768 [wandb_run.py:_redirect():1414] Wrapping output streams.
2021-04-25 23:32:08,767 INFO    MainThread:1768 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-25 23:32:08,768 INFO    MainThread:1768 [wandb_init.py:init():546] run started, returning control to user process
2021-04-25 23:32:09,047 INFO    MainThread:1768 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-25 23:32:09,047 INFO    MainThread:1768 [wandb_run.py:_restore():1466] restore
2021-04-25 23:32:09,267 INFO    MainThread:1768 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 585
  total_bytes: 585
}

2021-04-25 23:32:10,065 INFO    MainThread:1768 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 585
  total_bytes: 2918
}

2021-04-25 23:32:10,404 INFO    MainThread:1768 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2918
  total_bytes: 2918
}

2021-04-25 23:32:10,513 INFO    MainThread:1768 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2918
  total_bytes: 2918
}

2021-04-25 23:32:10,625 INFO    MainThread:1768 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2918
  total_bytes: 2918
}

2021-04-25 23:32:10,734 INFO    MainThread:1768 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2918
  total_bytes: 2918
}

2021-04-25 23:32:10,846 INFO    MainThread:1768 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2918
  total_bytes: 2918
}

2021-04-25 23:32:10,953 INFO    MainThread:1768 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2918
  total_bytes: 2918
}

